lib/workers/global/limits.ts

```diff
@@ -0,0 +1,40 @@
+import { logger } from '../../logger';
+
+const limitsToInit = ['prCommitsPerRunLimit'];
+const l = {};
+const v = {};
+
+export function init() {
+  let i;
+  for (i = 0; i < limitsToInit.length; i += 1) {
+    if (process.env[limitsToInit[i]]) {
+      setLimit(limitsToInit[i], parseInt(process.env[limitsToInit[i]], 10));
```

Comment:
The default value for this if not set should not be 10.  It should be treated as unlimited.